### PR TITLE
Update docs and location-api/sample.sh:

### LINKED ISSUE #365: Review and update tutorial docs
Priority: Low

## Issue Description

Review all docs and tutorials to ensure that changes made here https://github.com/Kuadrant/kcp-glbc/pull/356, haven't invalidated any docs or tutorials. 

I quickly scanned the docs, and updated where things were obviously wrong but i didn't go through the tutorial, and any screenshots that might be in that are probably out of date (screenshots might not be the best idea tbh).

### PR DESCRIPTION
**Pull request by ChristianZaccaria**
Closes #365 

## Description of Changes
Update docs and location-api/sample.sh:
- Make use of home workspace in tutorial doc.
- Remove compute and user workspace from Readme.
- Add GLBC_EXPORT to deployment doc.
- Update location-api/sample.sh to make it work with the home workspace.

### CODE REVIEW

File: docs/getting_started/tutorial.md [hunk 1]
```diff
@@ -142,7 +170,7 @@ The sample script will remain paused until we press the enter key to migrate the
 1. Verify that the DNS record was created:
    ```bash
    export KUBECONFIG=.kcp/admin.kubeconfig                                         
-   ./bin/kubectl-kcp workspace use root:kuadrant
+   ./bin/kubectl-kcp workspace use '~'
    kubectl get dnsrecords ingress-nondomain -o yaml
```
Comment (mikenairn) [inline docs/getting_started/tutorial.md:174]: This sample service has been updated, you will need to update the names of the resources it creates see https://github.com/Kuadrant/kcp-glbc/blob/main/samples/echo-service/echo.yaml

File: docs/getting_started/tutorial.md [hunk 2]
```diff
@@ -169,14 +197,12 @@ We will run the following commands in a new tab:
...
+   ./bin/kubectl-kcp workspace use '~'
    kubectl get ns default -o yaml
    ```
-As we can see, there is a label named: `*state.internal.workload.kcp.dev/kcp-cluster-1: Sync*`:
-
-   ![Screenshot from 2022-08-02 12-32-06](https://user-images.githubusercontent.com/73656840/182365628-22f04bb5-0818-46a3-8a12-3abc2e8451f3.png)
+As we can see, there is a label named something like: `*state.workload.kcp.dev/5MivhNIs7DjM7dK95I2K7TpWe7aUGMU4WHqjWn: Sync*`:
 
-GLBC is telling `kcp` where to sync all of the work resources in the namespace. Meaning, since the namespace is is set to `kcp-cluster-1` , the ingress will also have `kcp-cluster-1` set to it. 
+GLBC is telling `kcp` where to sync all of the work resources in the namespace. Meaning, since the namespace is set to `5MivhNIs7DjM7dK95I2K7TpWe7aUGMU4WHqjWn` (cluster-1) , the ingress will also have `5MivhNIs7DjM7dK95I2K7TpWe7aUGMU4WHqjWn` (cluster-1) set to it.
```
Comment (mikenairn) [inline docs/getting_started/tutorial.md:205]: This isn't the namespace, it's a hash of the workspace name i think.

File: docs/getting_started/tutorial.md [hunk 3]
```diff
@@ -169,14 +197,12 @@ We will run the following commands in a new tab:
...
-   ./bin/kubectl-kcp workspace use root:kuadrant
+   ./bin/kubectl-kcp workspace use '~'
    kubectl get ns default -o yaml
    ```
-As we can see, there is a label named: `*state.internal.workload.kcp.dev/kcp-cluster-1: Sync*`:
-
-   ![Screenshot from 2022-08-02 12-32-06](https://user-images.githubusercontent.com/73656840/182365628-22f04bb5-0818-46a3-8a12-3abc2e8451f3.png)
+As we can see, there is a label named something like: `*state.workload.kcp.dev/5MivhNIs7DjM7dK95I2K7TpWe7aUGMU4WHqjWn: Sync*`:
 
-GLBC is telling `kcp` where to sync all of the work resources in the namespace. Meaning, since the namespace is is set to `kcp-cluster-1` , the ingress will also have `kcp-cluster-1` set to it.
```
Comment (mikenairn) [inline docs/getting_started/tutorial.md:179]: Why did you remove kcp- from the name here?

File: docs/getting_started/tutorial.md [hunk 4]
```diff
@@ -217,29 +243,29 @@ Now that the DNS record has been successfully created, in a new tab in the termi
 
    ![Screenshot from 2022-08-02 12-44-15](https://user-images.githubusercontent.com/73656840/182368772-8a08a197-66d9-4d9c-9747-74ddaad0e4d7.png)
 
-This means that `kcp-cluster-1` is up and running correctly.
+This means that cluster-1 is up and running correctly.
```
Comment (mikenairn) [inline docs/getting_started/tutorial.md:246]: why cluster-1 and not kcp-cluster-1?

File: docs/getting_started/tutorial.md (hunk 1 above)
Comment (ChristianZaccaria) [reply docs/getting_started/tutorial.md]: Oh I didn't realize that. I just updated the names now to use "echo".

File: docs/getting_started/tutorial.md (hunk 4 above)
Comment (ChristianZaccaria) [reply docs/getting_started/tutorial.md]: I did this for simplicity in the tutorial I guess. But yes kcp- in the cluster name is what would be correct. I corrected that now :+1:

File: samples/location-api/sample.sh [hunk 5]
```diff
@@ -17,7 +17,6 @@
 #
 
 export KUBECONFIG=./.kcp/admin.kubeconfig
-BASE_WORKSPACE=root:kuadrant
```
[the remark this answers is not in the document]
Comment (ChristianZaccaria) [reply samples/location-api/sample.sh:20]: You're right, it's better that way :-1:

File: docs/getting_started/tutorial.md (hunk 2 above)
Comment (ChristianZaccaria) [reply docs/getting_started/tutorial.md]: I re-worded that. I meant to say that the `state` label in the namespace has that hash value set to it.

**Review by mikenairn — APPROVED** [review]
/lgtm :+1:
